### PR TITLE
Changes for new iOS SDK 13

### PR DESCRIPTION
**Pull request by Bjoe**
These changes are needed to use hunter with polly for the newest iOS SDK 13.

### CODE REVIEW

**Comment by Bjoe** [comment]
If you will merge my changes, please squash my commits. Thanks.

File: flags/ios_nocodesign.cmake
Comment (rbsheth) [inline flags/ios_nocodesign.cmake:32]: Nit: fix spacing

File: flags/ios_nocodesign.cmake
Comment (rbsheth) [inline flags/ios_nocodesign.cmake:10]: Do you also need to check the CMake version here?

File: flags/ios_nocodesign.cmake
Comment (Bjoe) [reply flags/ios_nocodesign.cmake:32]: Is fixed.

File: flags/ios_nocodesign.cmake
Comment (Bjoe) [reply flags/ios_nocodesign.cmake:10]: No, because [`CMAKE_XCODE_ATTRIBUTE_<an-attribute>`](https://cmake.org/cmake/help/v3.1/variable/CMAKE_XCODE_ATTRIBUTE_an-attribute.html) exists since cmake 3.1.

My change is only set an additional XCode property `CODE_SIGN_IDENTITY ""` to create a binary/library without signing it. It is neede since iOS sdK 13. Before it was enough to set only `CODE_SIGNING_REQUIRED NO`.

Btw. This works also for "older" xcode. My assumption is, we didn't need this `XCODE_XCCONFIG_FILE` approach ... but I'm not sure why this approach is introduced. It is enough to set the XCode propery via `CMAKE_XCODE_ATTRIBUTE_<an-attribute>` -->> `set(CMAKE_XCODE_ATTRIBUTE_CODE_SIGNING_REQUIRED NO)`

**Comment by Bjoe** [comment]
Sorry again.... please wait before merge. 
There is a new iOS sdk 13.1 out. Let me test this SDK with my changes. If all is fine, I let you know and then we can merge. Give me some days ...

**Comment by rbsheth** [comment]
@Bjoe Okay, waiting on your go-ahead

**Comment by Bjoe** [comment]
Ok, It works also with iOS 13.1 SDK.

**Review by rbsheth — APPROVED** [review]

**Pull request merged**
